mdx-models/src/main/java/com/mx/path/model/mdx/model/id/Authentication.java

```diff
@@ -25,6 +26,7 @@ public class Authentication extends MdxBase<Authentication> {
   private String deviceOperatingSystemVersion;
   private Double deviceLatitude;
   private Double deviceLongitude;
+  private DeviceRegistration deviceRegistration;
```

Comment:
Are we planning to do versioning going forward for all the new features we are going to add. Just curious since it is not a breaking change?

Reply:
No. This change will apply to both versions. So, migrating to the new version will not drop this field.